app/scripts/controllers.js

```diff
@@ -2,6 +2,7 @@
 
 /* Controllers */
 
+var site = 'http://localhost:8080';
```

Comment:
why is this hardcoded to localhost?

Reply:
right now that's the website URL but we can change it when we push to the appengine site.

Reply:
I am thinking there must be a deterministic way to figure that out. Since the js is downloaded from the backend.

Reply:
you're right, I fixed this